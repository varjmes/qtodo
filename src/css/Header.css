.header {
  margin: 0;
  padding: 2em;
  background: #ffe5e5;
}

.header__title {
  margin: 0;
  font-size: 4em;
}

.header__subtitle {
  display: block;
  font-size: 0.5em;
}
