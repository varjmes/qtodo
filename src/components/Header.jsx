import React from "react"
import "../css/Header.css"

export const Header = () => (
  <header className="header">
    <h1 className="header__title">
      qtodo
      <span className="header__subtitle">do things, it's cute</span>
    </h1>
  </header>
)

export default Header
