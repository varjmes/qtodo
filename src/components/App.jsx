import React, { Component, Fragment } from "react"
import "../css/App.css"

import Header from "./Header"
class App extends Component {
  render() {
    return (
      <Fragment>
        <Header />
        <p className="App-intro">
          To get started, edit <code>src/App.js</code> and save to reload.
        </p>
      </Fragment>
    )
  }
}

export default App
