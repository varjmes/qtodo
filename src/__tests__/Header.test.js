import React from "react"
import renderer from "react-test-renderer"
import { Header } from "../components/Header"

describe("<Header />", () => {
  test("should match the snapshot", () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
