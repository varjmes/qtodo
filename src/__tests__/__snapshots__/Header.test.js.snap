// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match the snapshot 1`] = `
<header
  className="header"
>
  <h1
    className="header__title"
  >
    qtodo
    <span
      className="header__subtitle"
    >
      do things, it's cute
    </span>
  </h1>
</header>
`;
